app/assets/stylesheets/components/header.scss

```diff
@@ -258,3 +254,18 @@ header {
     }
   }
 }
+
+  .menu--level-1,
+  .logo__pul {
+    height: 40px;
+    position: relative;
+  }
+
+  .menu--level-1>ul {
+    display: inline-block;
+
+    @media (min-width: 645px) {
+      margin-top: 0;
+      @include vertical-align;
+    }
+  }
```

Comment:
do we still need the changes in this file with the lux header?

Reply:
We don't need the ul bit, but do need to fix the height

Reply:
Fixed.